BaseUI/src/components/ImageCellFooter.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {Image, View} from 'react-native';
+import {Text} from 'react-native-elements';
+import {TouchableOpacity} from 'react-native-gesture-handler';
+import {Queue} from 'react-native-spacing-system';
+import {imageCell} from '../../styles/imageCell';
+
+interface ImageCellFooterProps {
+  likes?: number;
+  liked?: boolean;
+  callback: () => void;
+}
+
+const ImageCellFooter = (props: ImageCellFooterProps) => {
+  return (
+    <View style={imageCell.footerContainer}>
+      <Queue size={25} />
+      <TouchableOpacity
+        style={imageCell.likeButtonTouchableOpacity}
+        onPress={props.callback}>
+        <Image
+          source={require('../images/likes.png')}
+          style={props.liked ? imageCell.likeButton : imageCell.unlikeButton}
```

Comment:
```suggestion
          style={props.liked ? imageCellStyle.likeButton : imageCellStyle.unlikeButton}
```

а то не понятно, что это за `imageCell`